src/inputs/labeled-inputs.scss

```diff
@@ -8,6 +8,7 @@
   display: flex;
   flex-direction: column;
   flex-wrap: nowrap;
+  position: relative;
```

Comment:
Inline style has this rule already:
![image](https://user-images.githubusercontent.com/36186912/116847320-fbcec700-abf2-11eb-85e6-9c7bc5231169.png)

Reply:
Removed `position: relative` from `.iui-inline` and now only have it in the root of the SCSS file.